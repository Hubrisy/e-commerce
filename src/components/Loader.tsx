export const Loader: React.FC = () => (
  <div className="w-fit mx-auto">Loading...</div>
);
