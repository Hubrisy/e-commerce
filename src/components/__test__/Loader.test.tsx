import { render } from '@testing-library/react';

import { Loader } from '../Loader';

describe('Loader', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
  });
});
