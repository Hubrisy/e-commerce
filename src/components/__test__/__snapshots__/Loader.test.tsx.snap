// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<DocumentFragment>
  <div
    class="flex justify-center items-center h-24"
  >
    <div
      class="w-10 h-10 border-4 border-black border-t-transparent rounded-full animate-spin"
    />
  </div>
</DocumentFragment>
`;
